src/components/organisms/DataTable/DataTable.tsx

```diff
@@ -1,4 +1,11 @@
-import { CSSProperties, ReactElement, Ref, forwardRef, useState } from 'react'
```

Comment:
**[blocker]**

With the change to ApiClient (returning response.data instead of response.data.data) some places currently broke. Mostly you should also go over the mutation queries we do. (useUpdateRole, useCreateRole, useDeleteRole). The data there is now stored under ['roles', roleId] key. in the onSuccess callback we change the internal state of react-query, so we would have correct information there.

The current logic there assumes that the correct data for a user is the "data" returned from ApiClient, which used to be the case. However the "data" is no longer the same, so it is currently not updating the internal state correctly